.github/workflows/nb2post.yaml

```diff
@@ -0,0 +1,42 @@
+name: Notebook to Blog Post
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '_notebooks/*.ipynb'
+
+jobs:
+  convert-notebook:
+    if: contains(join(github.event.commits.*.message), '/sync')
```

Comment:
FYI this is one of the coolest parts about Actions which allow you to introspect the metadata of the events that trigger workflows.  In this line, we are checking to see if any of the commit messages in the push that triggers the workflow contain the string `/sync`